shopify_python/google_styleguide.py

```diff
@@ -0,0 +1,65 @@
+import importlib
+
+import astroid  # pylint: disable=unused-import
+
+from pylint import checkers, interfaces
+
+
+def register_checkers(linter):  # type: (astroid.ImportFrom) -> None
+    """Register checkers."""
+    linter.register_checker(GoogleStyleGuideChecker(linter))
+
+
+class GoogleStyleGuideChecker(checkers.BaseChecker):
+    """
+    Pylint checker for the Google Python Style Guide.
+
+    See https://google.github.io/styleguide/pyguide.html
+    """
+    __implements__ = (interfaces.IAstroidChecker,)
+
+    name = 'google-styleguide-checker'
+
+    msgs = {
+        'C9901': ('Imported an object that is not a package or module',
```

Comment:
The [pylint docs on custom checkers state](https://pylint.readthedocs.io/en/latest/reference_guide/custom_checkers.html):
> The message id should be a 5-digits number, prefixed with a message category. There are multiple message categories, these being C, W, E, F, R, standing for Convention, Warning, Error, Fatal and Refactoring. The rest of the 5 digits should not conflict with existing checkers and they should be consistent across the checker. For instance, the first two digits should not be different across the checker.